Makefile

```diff
@@ -798,6 +798,12 @@ openstack_update_run:
 	$(eval $(call vars,$@,openstack))
 	bash scripts/openstack-update.sh
 
+OV := $(shell oc get openstackversion -n $(NAMESPACE) -o name)
+.PHONY: openstack_patch_version
+openstack_patch_version: ## patches the openstackversion target version to the available version, if there is an update available
+	$(eval $(call vars,$@,openstack))
+	oc wait -n ${NAMESPACE} ${OV} --for=condition=MinorUpdateAvailable --timeout=${TIMEOUT} && \
```

Comment:
I'm not sure we strictly need this new condition to implement upgrade testing. If we add it we'll have it on main, but not on the FR1 maintenance branch so we won't be able to use the job there.

Reply:
It is enough to just check that the new catalog/index is deployed and that OpenStackVersion is in a ReadyCondition True state. Then if targetVersion != availableVersion you'd know there is an update (and this is shown clearly on the CLI too)

Reply:
as mentioned in the other PR, it just makes it a lot easier and it we can backport it. I'd say its a simple change and could be useful for any automation (also done by customers)